app/models/dialog.rb

```diff
@@ -89,7 +89,7 @@ def validate_field_data
   end
 
   def load_values_into_fields(values, overwrite = true)
-    values = values.with_indifferent_access
+    values = values.with_indifferent_access if values.kind_of?(Hash)
```

Comment:
What else could `values` be if not a hash?

Reply:
It's a string in the api tests.

Reply:
That sounds wrong to me, especially when `[]` is called on it [here](https://github.com/ManageIQ/manageiq/pull/17973/files#diff-ab119f9583edfc24f96ecba56bc8b0e5R96)

Reply:
Sounds like the tests need to be fixed